nebula_hw_interfaces/include/nebula_hw_interfaces/nebula_hw_interfaces_continental/continental_types.hpp

```diff
@@ -0,0 +1,152 @@
+  std::string characteristic_speed_status;
+  std::string radar_status;
+  std::string voltage_status;
+  std::string temperature_status;
+  std::string blockage_status;
+
+  ContinentalARS548Status() {}
+
+  /// @brief Stream ContinentalRadarStatus method
+  /// @param os
+  /// @param arg
+  /// @return stream
+  friend std::ostream & operator<<(std::ostream & os, nebula::ContinentalARS548Status const & arg)
+  {
+    os << "timestamp_nanoseconds: " << arg.timestamp_nanoseconds;
+    os << ", ";
+    os << "timestamp_seconds: " << arg.timestamp_seconds;
+    os << ", ";
+    os << "timestamp_sync_status: " << arg.timestamp_sync_status;
+    os << ", ";
+    os << "sw_version_major: " << arg.sw_version_major;
+    os << ", ";
+    os << "sw_version_minor: " << arg.sw_version_minor;
+    os << ", ";
+    os << "sw_version_patch: " << arg.sw_version_patch;
+    os << ", ";
+    os << "longitudinal: " << arg.longitudinal;
+    os << ", ";
+    os << "lateral: " << arg.lateral;
+    os << ", ";
+    os << "vertical: " << arg.vertical;
+    os << ", ";
+    os << "yaw: " << arg.yaw;
+    os << ", ";
+    os << "pitch: " << arg.pitch;
+    os << ", ";
+    os << "plug_orientation: " << arg.plug_orientation;
+    os << ", ";
+    os << "length: " << arg.length;
+    os << ", ";
+    os << "width: " << arg.width;
+    os << ", ";
+    os << "height: " << arg.height;
+    os << ", ";
+    os << "wheel_base: " << arg.wheel_base;
+    os << ", ";
+    os << "max_distance: " << arg.max_distance;
+    os << ", ";
+    os << "frequency_slot: " << arg.frequency_slot;
+    os << ", ";
+    os << "cycle_time: " << arg.cycle_time;
+    os << ", ";
+    os << "time_slot: " << arg.time_slot;
+    os << ", ";
+    os << "hcc: " << arg.hcc;
+    os << ", ";
+    os << "power_save_standstill: " << arg.power_save_standstill;
+    os << ", ";
+    os << "sensor_ip_address0: " << arg.sensor_ip_address0;
+    os << ", ";
+    os << "sensor_ip_address1: " << arg.sensor_ip_address1;
+    os << ", ";
+    os << "configuration_counter: " << arg.configuration_counter;
+    os << ", ";
+    os << "status_longitudinal_velocity: " << arg.longitudinal_velocity_status;
+    os << ", ";
+    os << "status_longitudinal_acceleration: " << arg.longitudinal_acceleration_status;
+    os << ", ";
+    os << "status_lateral_acceleration: " << arg.lateral_acceleration_status;
+    os << ", ";
+    os << "status_yaw_rate: " << arg.yaw_rate_status;
+    os << ", ";
+    os << "status_steering_angle: " << arg.steering_angle_status;
+    os << ", ";
+    os << "status_driving_direction: " << arg.driving_direction_status;
+    os << ", ";
+    os << "characteristic_speed: " << arg.characteristic_speed_status;
+    os << ", ";
+    os << "radar_status: " << arg.radar_status;
+    os << ", ";
+    os << "temperature_status: " << arg.temperature_status;
+    os << ", ";
+    os << "voltage_status: " << arg.voltage_status;
+    os << ", ";
+    os << "blockage_status_rate: " << arg.blockage_status;
```

Comment:
"blockage_status_rate" should probably be called "blockage_status" here, right?

Reply:
Thanks !
Addressed in [2338af7](https://github.com/tier4/nebula/pull/107/commits/2338af727c305890391d6daa356c55319e204dc5)